### PR TITLE
WEB-9133

### PR DESCRIPTION
**Pull request by joohuilee**
# WEB-9133 Migrate serato/slimulator to Github Actions

https://serato.atlassian.net/browse/WEB-9133

# Changes made

- New yaml file in githubs workflows that includes checks required
- Deleted travis file
- Fixed indentation in phpstan.neon
- Commented out phpstan test due to 95 errors

### CODE REVIEW

File: .github/workflows/automated-tests.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# This workflow is triggered every time a PR is opened, reopened, or the head branch of the pull request is updated.
+# Also is triggered when there is a push to master or a release branch. # It is also possible to trigger it manually.
+# ./vendor/bin/phpstan analyse is left out from Run Static Code Analyzers due to 95 errors
+name: Automated tests and static code analyzers
```
Comment (ben-farrelly) [inline .github/workflows/automated-tests.yml:4]: This doesn't run any automated tests in the way we term automated tests, maybe rename this to `Code checking and unit tests`

File: phpstan.neon [hunk 2]
```diff
@@ -3,5 +3,5 @@ parameters:
     paths:
         - %currentWorkingDirectory%/src
         - %currentWorkingDirectory%/tests
-    includes:
-        - vendor/phpstan/phpstan-phpunit/extension.neon
\ No newline at end of file
+includes:
+    - vendor/phpstan/phpstan-phpunit/extension.neon
```
Comment (ben-farrelly) [inline phpstan.neon:7]: 🥲

**Review by ben-farrelly — COMMENTED** [review]
Just one small naming thing, then this is good to go 😄

**Review by ben-farrelly — APPROVED** [review]

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        uses: php-actions/composer@v6
+        with:
+          php_version: ${{ matrix.php-versions }}
+          php_extensions: gmp
+          version: 2.2
+
+      - name: Run static code analyzers
+        run: |
+          ./vendor/bin/phpcs
+          ./vendor/bin/security-checker security:check
```
Comment (luke-wyeth) [inline .github/workflows/build.yml:47]: you could add a newline here between these two steps to match the convention of the rest of the file 😄

**Review by luke-wyeth — APPROVED** [review]
nice, this makes sense to me!

**Pull request merged**